crates/miden-tx/Cargo.toml

```diff
@@ -26,6 +26,7 @@ miden-objects = { workspace = true }
 # Miden dependencies
 miden-processor = { workspace = true }
 miden-prover    = { workspace = true }
+miden-stdlib    = { workspace = true }
```

Comment:
I think we don't need `miden-stdlib` dependency here, because it is already reexported as [`miden_lib::StdLib`](https://github.com/0xMiden/miden-base/blob/23a7be3b62f1d65a4fc89e207f644470f4780d70/crates/miden-lib/src/lib.rs#L30).